pkg/core/localize/locales/en/cmd/dedicated.en.toml

```
@@ -0,0 +1,42 @@
+[dedicated.registerCluster.cmd.shortDescription]
+one = 'Register an OpenShift cluster with Red Hat Streams for Apache Kafka'
```

We'd prolly like it to change in rest of mentions?
```suggestion
one = 'Register an OpenShift cluster with Red Hat OpenShift Streams for Apache Kafka'
```